docs/Architecture.md

```diff
@@ -190,6 +190,7 @@ Interfaces with cloud service providers or provides similar services locally.
     - `get_instance_public_ip(instance_id=None)`
     - `get_instance_local_ip(instance_id=None)`
     - `publish_stats(stat_name, stat_unit, stat_value)`: Record performance/load statistics.
+    - `publish_stats_multi(stat_name, stat_unit, stat_value)`: Record multiple performance/load statistics.
```

Comment:
parameter to `publish_stats_multi` should be just `stats`, a list of tuples, no?